src/test/java/hw2/ex2/SecondScenarioTest.java

```diff
@@ -0,0 +1,278 @@
+        driver.manage().timeouts().
+                pageLoadTimeout(20000, TimeUnit.MILLISECONDS);
+        driver.manage().timeouts().
+                setScriptTimeout(30000, TimeUnit.MILLISECONDS);
+    }
+
+    @Test
+    public void secondScenarioTest() {
+        // 1. Open test site by URL
+        // There's a simple opening of a site
+        driver.get("https://epam.github.io/JDI/index.html");
+
+        // 2. Assert Browser title
+        // There's a getting of a title and
+        // comparing it with expected title -- "Home page"
+        // NB! Soft Assert can be placed here 'cause we can
+        // continue test even if actual title isn't
+        // equal to expected
+        String actualTitle = driver.getTitle();
+        sa.assertEquals(actualTitle, "Home Page");
+
+        // 3. Perform login
+        // There we have a simple login steps:
+        // - clicking on icon
+        // - entering username and password
+        // - clicking on a login-button
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys("epam");
+        driver.findElement(By.id("password")).sendKeys("1234");
+        driver.findElement(By.id("login-button")).click();
+
+        // 4. Assert User name in the left top side
+        // of screen that user is logined
+        // There we have a getting of username and
+        // comparing it with expected -- "PITER CHAILOVSKII"
+        // (We should compare it as is - even if it's
+        // written in upper case)
+        WebElement actualUserName = driver.findElement(By.id("user-name"));
+        assertEquals(actualUserName.getText(), "PITER CHAILOVSKII");
+
+        // 5. Click on "Service" subcategory in
+        // the header and check that drop down contains options
+        // There is a clicking on "Service" header and getting actual
+        // header texts from service-dropdown menu. Then we create
+        // a list of expected text and compare them
+        driver.findElement(By.xpath("//li[@class='dropdown']")).click();
+        WebElement headerDropDownList = driver.findElement(By.
+                xpath("//ul[@class='dropdown-menu']"));
+        List<String> actualServiceHeaderTexts = Arrays.
+                asList(headerDropDownList.getText().split("\n"));
+        ArrayList<String> expectedServiceHeaderTexts = new ArrayList<>();
+        expectedServiceHeaderTexts.add("SUPPORT");
+        expectedServiceHeaderTexts.add("DATES");
+        expectedServiceHeaderTexts.add("SEARCH");
+        expectedServiceHeaderTexts.add("COMPLEX TABLE");
+        expectedServiceHeaderTexts.add("SIMPLE TABLE");
+        expectedServiceHeaderTexts.add("USER TABLE");
+        expectedServiceHeaderTexts.add("TABLE WITH PAGES");
+        expectedServiceHeaderTexts.add("DIFFERENT ELEMENTS");
+        expectedServiceHeaderTexts.add("PERFORMANCE");
+        assertEquals(actualServiceHeaderTexts, expectedServiceHeaderTexts);
```

Comment:
1. Creation of the List could be simplified
2. Using split("\n") not good idea(

Reply:
Solved:
1. I use *Arrays.asList* now, it looks more simple
2. I get all <li> from that list now and added them to my ArrayList